### ISSUE TITLE
y transformed to u?

### ISSUE THREAD

**Issue by PonteIneptique** [avian2/unidecode#22]
Hi there !
I hope this is really a bug and not me not being sure of what's going on... Anyway.

## Context

- `unidecode==0.4.21`
- python 3.5.2
- Linux Mint Sonya 18.2 (Not that I think it matters here)

## Reproduction of the bug

```python
from unidecode import unidecode
print(unidecode("y"))  # prints y
print(unidecode("ў"))  # prints u
```

## Expected
`print(unidecode("ў"))` should print `y` as well

I'll try to see if I can understand the reason why it works this way in the codebase but at least the issue is open...

**Comment by PonteIneptique** [comment]
The same happens with `unidecode_expect_nonascii`

**Comment by avian2** [comment]
"ў" is Cyrillic letter "u" (U+045E CYRILLIC SMALL LETTER SHORT U).

From [Wikipedia](https://en.wikipedia.org/wiki/U_(Cyrillic)):

> It is normally romanised as "u", but in Kazakh, it is romanised as "w".

**Comment by PonteIneptique** [comment]
Well, It's certainly interesting... This is generally used in literature studies to mark short voyels (in this case y). Sorry for opening a bug then :)

**Comment by avian2** [comment]
If it is supposed to be a Latin letter "y" with a breve diacritic in your text ("y̆"), then I think it should be encoded as U+0079 U+0306 (Latin letter y, followed by a combining breve character). Unicode doesn't provide a canonical composition in this case (a single code point - like it does for the Cyrilic U+045E), but the decomposed form should still be valid. In that case Unidecode will correctly transliterate "y̆" into "y".

So in short, I believe this is a problem with the input you are giving Unidecode, not Unidecode itself.
